src/styles/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    border-radius: 1px;
```

```suggestion
    border-radius: 1px;
```
не до конца понял для чего здесь бордер-радиус?

---

то на майбутнє) якшо в них з'явиться ховер ) а тако ж це бездумне копіювання з фігми ))

---

понял)

---

я б видалив, в майбутньому коли знадобиться, тоді й додаси